cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -126,12 +138,21 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
   std::vector<ceph::bufferlist*> chunks_;
 };
 
+/// \brief Scan RADOS objects containing Parquet binary data.
+/// \param[in] hctx RADOS object context.
+/// \param[in] filter The filter expression to apply.
+/// \param[in] partition_expression Partition expression.
+/// \param[in] projection_schema The projection schema.
+/// \param[in] dataset_schema The dataset schema.
+/// \param[out] table Table to store the resultant data.
+/// \param[in] file_size File size.
```

Comment:
Same as the comments below about the filter and partition expression parameters

Reply:
Done.